mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -402,6 +402,35 @@ def push_boefje_task(
             task_db.status = models.TaskStatus.FAILED
             self.ctx.datastores.task_store.update_task(task_db)
 
+        # Is the ooi in other organisations? When the ooi is in other
+        # organisations we need to create tasks for those organisations as well.
+        if boefje_task.input_ooi is not None:
+            # FIXME: is the ooi shared between organisations, or is it a copy?
+            # If it is a copy we need to use the ooi id in the boefje task.
+            orgs = self.is_ooi_in_other_organisations(boefje_task.input_ooi)
+            if orgs:
+                boefje_task.organizations = orgs
+
+                # FIXME: what if boefje is disabled in the other org? Are we sure
+                # that in that org the boefje is allowed to scan the ooi?
+                # FIXME: what status do we give it do we give it completed, we
```

Comment:
I'd say a new state Deduplicated would fit the bill? We could then change it to 'complete' or 'error' once the actual running job completes/errors.